Source/WebCore/html/HTMLCanvasElement.h

```diff
@@ -153,6 +153,8 @@ class HTMLCanvasElement final : public HTMLElement, public CanvasBase, public Ac
     void eventListenersDidChange() final;
 
     void parseAttribute(const QualifiedName&, const AtomString&) final;
+    bool hasPresentationalHintsForAttribute(const QualifiedName&) const override;
+    void collectPresentationalHintsForAttribute(const QualifiedName&, const AtomString&, MutableStyleProperties&) override;
```

Comment:
ditto

Reply:
Made the change. Methods should now be final